Scripts/Set-Output.ps1

```diff
@@ -8,4 +8,4 @@ param (
     $Value
 )
 
-return Write-Output "$Key=$Value" >> $GITHUB_OUTPUT
+Write-Output $Key=$Value >> $GITHUB_OUTPUT
```

Comment:
Are you sure the quotes are not needed around `$Key=$Value`? Also, is `Write-Output` really needed, isn't just writing to the variable enough?

Reply:
Yes, It is working as expected without the quotes, and I will check on the `Write-Output`.

Reply:
Yup, the `Write-Output` is unnecessary.